Source/Core/AudioCommon/NullSoundStream.cpp

```diff
@@ -2,12 +2,13 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include <cstring>
-
 #include "AudioCommon/NullSoundStream.h"
+#include "Common/CommonTypes.h"
 #include "Core/HW/AudioInterface.h"
 #include "Core/HW/SystemTimers.h"
 
+constexpr size_t NullSound::BUFFER_SIZE;
```

Comment:
Not behavior-wise as it is, however I left it in because without it, if BUFFER_SIZE is ever used in a clamp or min/max call in the future, it won't randomly fail to compile, as there wouldn't be an address to get the value at (min/max/clamp all take arguments as references, and it's required to satisfy the one definition rule in that case). However I'll remove it because in retrospect, it doesn't make sense to do that in a null audio interface.